README.md

```diff
@@ -1 +1,6 @@
-# Contact-Form
\ No newline at end of file
+# Contact-Form
+# Description
+This is a contact form management system that enable user to send message to the site owner. The user fills in his/her name, email, subject of the message and the message. when the user click send message button the message is saved in the database. The site owner can view all message on a table in the view_contact.php page.
+
```

Comment:
Your README.md isn't clear enough.

This application doesn't work on my local machine till I tweak it. My point? You're the author of the code, you know how it works cause you structured it. Make sure your README.md file contains instructions/steps for usage and not just a summary.

Reply:
Okay i understand now